src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -241,6 +241,12 @@ public override void Execute(DocumentsOperationContext context)
                             cmd.Document.TryGet(Constants.Documents.Metadata.Key, out metadata);
                             LastEtag = putResult.Etag;
 
+                        
+                            metadata.Modifications = new DynamicJsonValue(metadata)
```

Comment:
This seems redundant? Why we are sending Etag/Id back inside metadata also? Do we need to send metadata even? I do not see any modifications inside Put?